lib/assembly/schemas/build.ex

```diff
@@ -14,6 +14,8 @@ defmodule Assembly.Schemas.Build do
     field :order_id, :string
     field :status, BuildStatusEnum
 
+    field :missing_components, :any, virtual: true
```

Comment:
Can we switch this to be a bit more typed? It makes it a bit easier to read when looking at the schema.
```
field :missing_components, {:array, :string}, virtual: true
```

Reply:
It's not a string, so it would need to be `:map` but it's actually a struct, but not an association since it's virtual. Therefore `:any` is the only way to skip type checking which we're not looking for.

Reply:
hmmm, that's annoying. I'm assuming you can't do `{:map, %Bottle...}` so I guess this is the best we can do.